### PR TITLE
Changes to README.md

### PR DESCRIPTION
**Pull request by masaguaro**
Hi, these are some changes I made to README.db after discussing with Mani.. Feel free to make any comment or suggestion to improve them.

### CODE REVIEW

**Comment by neomatrix369** [comment]
@masaguaro good work, I'll post my comments, I suggest you install Grammarly plugin (https://www.grammarly.com) on your browser it will help correct any future mistakes - I understand English isn't our first language

File: README.md [hunk 1]
```diff
@@ -1,4 +1,46 @@
 # machine_learning_notes
+
 As part of ***Machine Learning Study Group*** from recworks meet-a-mentor community
 
-work on kaggle ML project: https://www.kaggle.com/c/house-prices-advanced-regression-techniques
+work on kaggle ML project.
```
Comment (neomatrix369) [inline README.md]: What is this a title or just a note?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,46 @@
...
+
+We use Jupyter Notebook, Python and some libraries (Pandas, NumPy, Matplotlib and Scikit-learn) to solve ML problems on public data sets. We have started with the following data set from Kaggle:
+
+https://www.kaggle.com/c/house-prices-advanced-regression-techniques.
```
Comment (neomatrix369) [inline README.md]: Dot at the end isn't required

File: README.md [hunk 3]
```diff
@@ -1,4 +1,46 @@
...
+
+This project is the result of the joined effort of a study group of enthusiastic people from different professional background. Anyone is welcome to join and to contribute with the discussion.
```
Comment (neomatrix369) [inline README.md]: joined => joint effort

Comment (neomatrix369) [reply README.md]: discussion ==> discussions

File: README.md [hunk 4]
```diff
@@ -1,4 +1,46 @@
...
-work on kaggle ML project: https://www.kaggle.com/c/house-prices-advanced-regression-techniques
+work on kaggle ML project.
+
+We use Jupyter Notebook, Python and some libraries (Pandas, NumPy, Matplotlib and Scikit-learn) to solve ML problems on public data sets. We have started with the following data set from Kaggle:
+
+https://www.kaggle.com/c/house-prices-advanced-regression-techniques.
+
+This project is the result of the joined effort of a study group of enthusiastic people from different professional background. Anyone is welcome to join and to contribute with the discussion. 
+
+The best way to join to the project is by assisting to on our regular session.  Also, feel free to fork and check what we have done in this project. We are very happy to receive any comment and suggestions to improve.
```
Comment (neomatrix369) [inline README.md]: to on our regular session ==> during our regular sessions

File: README.md [hunk 5]
```diff
@@ -1,4 +1,46 @@
...
+
+
+There are two notebooks contained in the repository:
+
+## 1) machine_learning_notes.ipynb:
+
+This notebook contains:
+
+* 1 - Define the problem
+* [Load data and displaying info](#load-data)
```
Comment (neomatrix369) [inline README.md]: Indentation to the right

File: README.md [hunk 6]
```diff
@@ -1,4 +1,46 @@
...
+    * Select features using random forest classifier #WIP
+    
+* 3 - [Spot Check Algorithms](#spot-check)
+    * split dataset
+    * train on multiple algorithms  # WIP
+    
+
+## 2) plot_outlier_detection.ipynb
+
+This notebook contains a reference for some techniques to tackle the problem to detect and remove outliers sample from the data set.
```
Comment (neomatrix369) [inline README.md]: all throughout the text, `dataset` is one word, and not `data set` try to remove the spaces where required

**Review by neomatrix369 — CHANGES_REQUESTED** [review]
LGTM - except a few comments left on the page

**Comment by neomatrix369** [comment]
Try to use grammarly to see if it helps improve the sentences otherwise its fine the way it is

File: README.md (hunk 4 above)
Comment (masaguaro) [reply README.md]: Does it sound better? 'The best way to join the project is by assisting during our regular sessions.'

**Comment by neomatrix369** [comment]
Merging nw, as agreed, any other pending stuff we do via a new PR

**Pull request merged**